spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/TraceHeaders.java

```diff
@@ -38,76 +38,88 @@
 @ConfigurationProperties("spring.sleuth.headers")
 public class TraceHeaders {
 
-	private Zipkin zipkin;
-	private Sleuth sleuth;
-
-	public Zipkin getZipkin() {
-		return this.zipkin;
+	public static final String SPAN_SAMPLED = "1";
+	public static final String SPAN_NOT_SAMPLED = "0";
+
+	public static final String ZIPKIN_TRACE_ID_HEADER_NAME = "X-B3-TraceId";
+	public static final String ZIPKIN_SPAN_ID_HEADER_NAME = "X-B3-SpanId";
+	public static final String ZIPKIN_PARENT_SPAN_ID_HEADER_NAME = "X-B3-ParentSpanId";
+	public static final String ZIPKIN_SAMPLED_HEADER_NAME = "X-B3-Sampled";
+	public static final String ZIPKIN_PROCESS_ID_HEADER_NAME = "X-Process-Id";
+
+	private String traceId = ZIPKIN_TRACE_ID_HEADER_NAME;
```

Comment:
I think these constant names are long winded and a bit imprecise.

Ex the _NAME suffix is wasted characters IMHO, and unnecessarily causes
line wraps.

There's a ZIPKIN_ header that has nothing to do with zipkin or b3 (process
ID)

Here's an example of the complete set of zipkin/b3 headers. My only
suggestion here would to be rename it to PropagationHeaders. Also, if you
do that, you'll notice you can drop the HEADERS part, too! The whole thing
is wildly more succinct

https://github.com/openzipkin/brave/blob/master/brave-http/src/main/java/com/github/kristofa/brave/http/BraveHttpHeaders.java